src/test/java/org/jsmart/zerocode/core/envvar/ZeroCodeEnvPropertyReaderTest.java

```diff
@@ -0,0 +1,20 @@
+package org.jsmart.zerocode.core.envvar;
+
+import org.jsmart.zerocode.core.domain.EnvProperty;
+import org.jsmart.zerocode.core.domain.JsonTestCase;
+import org.jsmart.zerocode.core.domain.TargetEnv;
+import org.jsmart.zerocode.core.tests.customrunner.TestOnlyZeroCodeUnitRunner;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+@TargetEnv("app_config.properties")
+@EnvProperty("${ENV_NAME}_") // see "ENV_NAME=ci" in .bash_profile file. If not found, then defaults to "app_config.properties"
+@RunWith(TestOnlyZeroCodeUnitRunner.class)
+public class ZeroCodeEnvPropertyReaderTest {
```

Comment:
yes, nice, good one !

Where is the similar test for package runner?
Can you please add an integration test for package/suite runner?

Reply:
Added a test. Look for `org.jsmart.zerocode.core.envprop.suite.ZeroCodeEnvPropertyReaderPackageTest`

Reply:
👍